webrtc.html

```diff
@@ -5236,6 +5240,10 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                   <var>sender</var>.</p>
                 </li>
                 <li>
+                  <p>If <var>connection</var>'s [[<a>isClosed</a>]] slot is
+                  <code>true</code>, abort these steps.</p>
+                </li>
+                <li>
```

Comment:
@adam-be My understanding is we don't resolve asynchronous operations on a closed peer connection. At least that's the case for `setLocalDescription` and `setRemoteDescription`.

E.g. step 2.1.1. in http://w3c.github.io/webrtc-pc/#set-description
> If connection's [[isClosed]] slot is true, then abort these steps.